staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -238,6 +238,26 @@ var (
 		[]string{"source", "status"},
 	)
 
+	requestTimestampComparisonDuration = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name:           "apiserver_request_timestamp_comparison_time",
```

Comment:
@logicalhan We're not necessarily planning on having these two metrics for a very long time, do you have any recommendations?

Reply:
use alpha as the metric stability level? and annotate it in the HELP description?

Reply:
Sounds good, thanks!